### ISSUE TITLE
Toggle: Accordion <summary> has role="tab" which is not allowed

### ISSUE THREAD

**Issue by szet0018** [wet-boew/wet-boew#8919]
https://wet-boew.github.io/v4.0-ci/demos/toggle/toggle-en.html#accordion

aXe accessibility checker is reporting the following issue:

**ARIA role must be appropriate for the element**

**Issue description**
Ensures role attribute has an appropriate value for the element
Impact: minor

**Element location**
`#wb-auto-11`

**Element source**
`<summary class="wb-toggle tgl-tab wb-init wb-toggle-inited" data-toggle="{&quot;parent&quot;: &quot;.accordion&quot;, &quot;group&quot;: &quot;.acc-group&quot;}" id="wb-auto-11" role="tab" aria-selected="false" tabindex="0" aria-posinset="1" aria-setsize="3">Example 1</summary>`

To solve this violation, you need to:
Fix the following:
ARIA role tab is not allowed for given element

No role is permitted on summary, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary
- Default role for summary is button

**Comment by duboisp** [comment]
I reviewed PR#8948 - I understand the issue, but all those "details/summary" is wrapped in a tabpanel role - https://github.com/wet-boew/wet-boew/blob/master/src/plugins/toggle/toggle.js#L127

That means we will also need to change the pattern used for tabs interface. Like here you can see that the toggle plugin is intentionally called - https://github.com/wet-boew/wet-boew/blob/master/src/plugins/tabs/tabs.js#L220-L222

So the solution will need to ensure the whole tabbing pattern is conforming, not only removing the role tab in the summary element.
